packages/open-api-gateway/src/project/open-api-gateway-ts-project.ts

```diff
@@ -131,6 +131,9 @@ export class OpenApiGatewayTsProject extends TypeScriptProject {
     });
     spec.synth();
 
+    // Delete the lib directory prior to compilation to ensure latest parsed spec json file is copied
+    this.preCompileTask.exec(`rm -rf ${this.libdir}`);
```

Comment:
Looks like my suggestion worked ;)

Reply:
It did indeed! Much more robust than copying the single file 🎉  Made a test project and updated the spec.yaml a bunch of times and it always rebuilt :)